docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -385,8 +385,8 @@ personRepository.deleteAll();
 // delete by id
 boolean deleted = personRepository.deleteById(personId);
 
-// update all living persons
-personRepository.update("name = 'Moral' where status = ?1", Status.Alive);
+// set all persons named "Mortal" to living.
```

Comment:
This is not what it does. It updates all living persons to change their name to `Mortal`. As a result, the mongodb equivalent example should probably be changed to reflect that.

Reply:
I update all comments and switch the MongoDB examples to mimic the one from Hibernate with Panache guide.